src/renderer/components/opds/BrowserResult.tsx

```diff
@@ -140,6 +151,12 @@ export class BrowserResult extends React.Component<BrowserResultProps, null> {
             return newUrl;
         }
     }
+
+    private goto(url: string) {
+        if (url) {
```

Comment:
`url` is "guaranteed" to be non-null/non-undefined (well, assuming TypeScript `strictNullChecks` compiler settings), so I would remove the `if (url) {}` conditional statement here.

Reply:
No it's not guaranteed. When the user use shortkeys, the function is called even if the url is undefined

Reply:
Oh ok, I just read this one before reply, I'll do as you say for the shortkeys